### PR TITLE
fixing e2e test for popup timing

### PR DESCRIPTION
**Pull request by mboudreau**
**What's in this PR?**
Trying to fix e2e test consistentcy

**Why**
Because it's been failing for a while and nobody was looking into it

**Added feature flags**
none

**Affected issues**  
None

**How has this been tested?**  
e2e test

**Whats Next?**
Make e2e tests required so this doesn't happen again

### CODE REVIEW

**Review by rachellerathbone — DISMISSED** [review]

**Review by krazziekay — DISMISSED** [review]

File: test/e2e/app-installation.e2e.ts [hunk 1]
```diff
@@ -1,64 +1,32 @@
...
+			expect(await jiraAppInstall(page)).toBeTruthy();
+		});
+	});
 
-				/*
-				test("githubAppInstall", async ({ page }) => {
-					expect(await githubAppInstall(page)).toBeTruthy();
-				});
+	// eslint-disable-next-line jest/expect-expect
+	test.describe.skip("github", () => {
```
Comment (krazziekay) [inline test/e2e/app-installation.e2e.ts:22]: Is this meant to be skipped?

File: test/e2e/login.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+					test.use({
+						storageState: testData.jira.roles.admin.state
+					});
+				}
+				test("jiraLogin", async ({ page }) => {
+					expect(await jiraLogin(page, "admin")).toBeTruthy();
+				});
+			});
+
+			test.describe.skip("Github", () => {
```
Comment (krazziekay) [inline test/e2e/login.e2e.ts:20]: Same here, whats the reason for skipping this?

File: test/e2e/setup.ts [hunk 3]
```diff
@@ -10,15 +9,16 @@ export default async function setup() {
 	const page = await browser.newPage();
 
 	// Remove old state before starting
-	clearState();
+	// clearState();
```
Comment (krazziekay) [inline test/e2e/setup.ts:12]: Can remove this from here.

File: test/e2e/app-installation.e2e.ts (hunk 1 above)
Comment (mboudreau) [reply test/e2e/app-installation.e2e.ts:22]: yes as it's not ready yet

File: test/e2e/login.e2e.ts (hunk 2 above)
Comment (mboudreau) [reply test/e2e/login.e2e.ts:20]: same as above

File: test/e2e/setup.ts (hunk 3 above)
Comment (mboudreau) [reply test/e2e/setup.ts]: actually, forgot to uncomment this :grimacing:

**Review by krazziekay — APPROVED** [review]

**Pull request merged**